wgsl/index.bs

```diff
@@ -41,11 +41,32 @@ thead {
 
 ## Goals ## {#goals}
 
- * Trivially convertable to SPIR-V
- * Constructs are defined as normative references to their SPIR-V counterparts
- * All features in [SHORTNAME] are directly translatable to SPIR-V. (No polymorphism, no general pointers, no overloads, etc)
- * Features and semantics are exactly the ones of SPIR-V
- * Each item in this spec *must* provide the mapping to SPIR-V for the construct
```

Comment:
> * Features and semantics are exactly the ones of SPIR-V
> * Each item in this spec *must* provide the mapping to SPIR-V for the construct

We seem to have lost these two goals, with in my opinion were pretty key to the entire WGSL proposal.